hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/yerlanov/hw01_hello_otus
```

```suggestion
module github.com/yerlanov/otus-homework/hw01_hello_otus
```
Название репозитория тоже следует использовать в полном имени модуля

---

Не заметил, спасибо!
Принял правку.